chef/cookbooks/ceph/recipes/osd.rb

```diff
@@ -69,7 +69,13 @@
   if is_crowbar?
     node.set["ceph"]["osd_devices"] = [] if node["ceph"]["osd_devices"].nil?
     min_size_blocks = node["ceph"]["osd"]["min_size_gb"] * 1024 * 1024 * 2
-    unclaimed_disks = BarclampLibrary::Barclamp::Inventory::Disk.unclaimed(node).sort.select { |d| d.size >= min_size_blocks }
+    claim_string = "Ceph"
+    # claim only disks with Ceph role; when there is none, take disks without role
+    ceph_disks, disks_without_role = BarclampLibrary::Barclamp::Inventory::Disk.prepared_for(node,claim_string)
+    unclaimed_disks = ceph_disks.empty? ? disks_without_role : ceph_disks
+    unclaimed_disks.sort.select! {|d|
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)